buildSrc/src/main/kotlin/Extensions.kt

```diff
@@ -120,6 +131,16 @@ fun Project.setupLibraryModule(block: LibraryExtension.() -> Unit = {}): Library
     }
 }
 
+fun Project.setupAppModule(block: BaseAppModuleExtension.() -> Unit = {}): BaseAppModuleExtension {
```

Comment:
I don't think we gain much by factoring this out of the `coil-sample` build file since it's the only app module.